### PR TITLE
feature: added update in place to improve user experience

### PR DESCRIPTION
**Pull request by ratogabriel**

### CODE REVIEW

File: appimage/utils.go [hunk 1]
```diff
@@ -520,3 +555,11 @@ func Remove(options types.RemoveOptions, config config.Store) error {
 
 	return bar.Finish()
 }
+
+func randomTempName() string {
```
[the remark this answers is not in the document]
Comment (ratogabriel) [reply appimage/utils.go:559]: yeah, on force remove flag, remember it? xD

**Comment by ratogabriel** [comment]
fixed test

File: appimage/utils.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (srevinsaju) [reply appimage/utils.go:559]: Can you point me to the line number where its still beiung used? I don't seem to find it

[the remark this answers is not in the document]
Comment (srevinsaju) [reply appimage/utils.go:559]: @ratogabriel would you like me to remove these, or do you want to continue on the same?

**Pull request merged**